daemon/cmd/status.go

```diff
@@ -599,16 +600,22 @@ func (d *Daemon) getStatus(brief bool) models.StatusResponse {
 
 	sr.Stale = stale
 
+	//CiliumVersion definition
+	ver := version.GetCiliumVersion()
+	ciliumVer := fmt.Sprintf("%s (v.%s-r.%s)", ver.Version, ver.Version, ver.Revision)
+
 	switch {
 	case len(sr.Stale) > 0:
+		msg := "Stale status data"
 		sr.Cilium = &models.Status{
 			State: models.StatusStateWarning,
-			Msg:   "Stale status data",
+			Msg:   fmt.Sprintf("%s    %s", ciliumVer, msg),
```

Comment:
it seems the space was not removed? As well as in the remaining lines.

Reply:
> it seems the space was not removed? As well as in the remaining lines.
Yes, it is. This is OK. You only mentioned the space between State and Version: 
> The space between the Ok and the [Ver: 1.9.90 Rev: 7ce4c3609] seems unnecessary. Also, to make it consistent maybe we should make it between parenthesis, similar to what is done for Kubernetes, and not square brackets.
But the space between Version and Msg is needed as a delimeter. Please, see comment from [sayboras](https://github.com/sayboras):
> while string concatenation is faster, it's hard to read and error prone (e.g. ciliumVer must have space prefix, msg should be ending with dot or comma, etc). I would prefer to go with fmt.Sprintf() in general.